examples/vic-gov-au/nuxt.config.js

```diff
@@ -112,7 +112,9 @@ export default {
   * Configuration for ripple-nuxt-ui
   * See https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-ui
   */
-  ripple: {},
+  ripple: {
+    customIcon: true
+  },
```

Comment:
updated so I can confirm locally that custom icon is still working

Reply:
This example is nice. Can you please move it to https://github.com/dpc-sdp/ripple/tree/master/examples/basic-examples ?